ccd-definition/ComplexTypes.json

```diff
@@ -1 +1,155 @@
-[]
+[
+  {
+    "ID": "SolicitorReferences",
+    "ListElementCode": "solicitorReference",
+    "FieldType": "Text",
+    "ElementLabel": "Claimant's legal representative's reference",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "SolicitorReferences",
+    "ListElementCode": "defendantReference",
+    "FieldType": "Text",
+    "ElementLabel": "Defendant's legal representative's reference",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "CourtLocation",
+    "ListElementCode": "preferredCourt",
+    "FieldType": "Text",
+    "ElementLabel": "Court name",
+    "HintText": "This should be your preferred County Court hearing centre.",
+    "SecurityClassification": "Public"
```

Comment:
Question, can there only be one party showing a preference for a court location? e.g. can each party provide a preference and then someone else decides based on that?

Reply:
Question for business I think!

Reply:
Business response - 
"the defendant get the chance to state their preferred court at the DQ stage, we play back the claimants preferred court and ask if they agree, if not they can provide their preferred court location.

If its two claimants it would be an agreed court between the two of them"

Given this, I would change the value from 'preferredCourt' to 'applicantPreferredCourt'

Reply:
Created https://tools.hmcts.net/jira/browse/CMC-489